packages/dev/src/main.test.ts

```diff
@@ -447,6 +447,61 @@ describe('Handling requests', () => {
       await dev.stop()
       await fixture.destroy()
     })
+
+    test('Image CDN requests are supported', async () => {
+      const fixture = new Fixture()
+        .withFile(
+          'netlify.toml',
+          `[images]
+         remote_images = [
+           "https://images.unsplash.com/photo-1517849845537.*"
```

Comment:
This will actually fetch the image from unsplash.com, right? Can we either mock the HTTP client or use a local URL to avoid a situation where the server is unavailable or the URL changes and tests start failing randomly?